HccAna/src/HccAna.cc

```diff
@@ -2605,8 +2605,8 @@ void HccAna::bookPassedEventTree(TString treeName, TTree *tree)
                                    std::vector<pat::Jet> selectedMergedJets,
                                    std::map<unsigned int, TLorentzVector> selectedFsrMap)*/
 void HccAna::setTreeVariables( const edm::Event& iEvent, const edm::EventSetup& iSetup,
-                                   std::vector<pat::Jet> goodJets, std::vector<float> goodJetQGTagger,
-                                   std::vector<float> goodJetaxis2, std::vector<float> goodJetptD, std::vector<int> goodJetmult,
+                                   std::vector<pat::Jet> goodJets, //std::vector<float> goodJetQGTagger,
```

Comment:
goodJetQGTagger, goodJetaxis2, goodJetptD, goodJetmult will not be used anymore?

Reply:
As far as I know, we were not using theme. We inherited them from the old code and I had to remove them because I got some errors in CMSSW_13